hack/validate-migration.sh

```diff
@@ -109,16 +150,21 @@ check_apply_on_pipelines() {
     local failed=
     local file_path=
     local updated=
+    local tmp_pl_file=
     while read -r file_path; do
-        cp "$file_path" "${file_path}.copy"
-        if ! bash -x "$migration_file" "${file_path}.copy" 2>"$run_log_file" >&2; then
-            echo "error: failed to run migration file $migration_file on pipeline $file_path:" >&2
+        tmp_pl_file="${file_path}.copy"
+        cp "$file_path" "$tmp_pl_file"
```

Comment:
From the name of variables I cannot conclude the second is a copy of the first. Please make them match, like `pl_file_path` and `pl_file_copy_path` or similar.